index.js

```diff
@@ -35,18 +50,18 @@ if (argv.gasprice < 1 || argv.gasprice > 1000) throw "--gasprice must be between
   const account = accounts[0];
   const networkId = await web3.eth.net.getId();
 
-  // Example EMP Parameters. Customize these.
+  // EMP Parameters. Pass in arguments to customize these.
   const empParams = {
-    expirationTimestamp: "1640995200", // 01/01/2022 @ 0:00 (UTC)
-    collateralAddress: "0xd0a1e359811322d97991e03f863a0c30c2cf029c", // Kovan WETH address. Mainnet WETH is: 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2.
-    priceFeedIdentifier: padRight(utf8ToHex("USDETH"), 64), // Using the USDETH price.
-    syntheticName: "uUSDwETH Synthetic Token Expiring 1 January 2022", // Long name.
-    syntheticSymbol: "uUSDwETH-JAN", // Short name.
+    expirationTimestamp: argv.expirationTimestamp.toString(), // Timestamp that the contract will expire at.
+    collateralAddress: argv.collateralAddress.toString(), // Collateral token address.
+    priceFeedIdentifier: padRight(utf8ToHex(argv.priceFeedIdentifier.toString()), 64), // Price identifier to use.
+    syntheticName: argv.syntheticName, // Long name.
+    syntheticSymbol: argv.syntheticSymbol, // Short name.
     collateralRequirement: { rawValue: toWei("1.25") }, // 125% collateral req.
     disputeBondPct: { rawValue: toWei("0.1") }, // 10% dispute bond.
-    sponsorDisputeRewardPct: { rawValue: toWei("0.05") }, // 5% reward for sponsors who are disputed invalidly
+    sponsorDisputeRewardPct: { rawValue: toWei("0.05") }, // 5% reward for sponsors who are disputed invalidly.
     disputerDisputeRewardPct: { rawValue: toWei("0.2") }, // 20% reward for correct disputes.
-    minSponsorTokens: { rawValue: toWei("100") }, // Min sponsor position size of 100 synthetic tokens.
+    minSponsorTokens: { rawValue: toWei(argv.minSponsorTokens.toString()) }, // Minimum sponsor position size.
```

Comment:
Do any of these need `.toString()`? I think telling minimist to interpret them as strings guarantees that they're either strings or undefined/null.

Reply:
I was getting an `invalid BigNumber value` error without `.toString()`, even when passing the args in as strings. Open to other suggestions on how to solve this.